tests/scrutinizer.dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM elabftw/elabimg:hypernext
+
+RUN apk upgrade -U -a && apk add --no-cache php8-simplexml
+
+# phpStan
+ADD https://github.com/phpstan/phpstan/releases/download/1.4.2/phpstan.phar /usr/bin/phpstan
```

Comment:
versions for things like that should be in VARS at top of file

Reply:
Yes, that makes a lot of sense.